netket/experimental/dynamics/_rk_solver_structures.py

```diff
@@ -323,7 +323,7 @@ def __post_init__(self):
             self.norm = euclidean_norm
 
         if self.dt_limits is None:
-            self.dt_limits = (None, 10 * self.initial_dt)
+            self.dt_limits = (0.01 * self.initial_dt, 10 * self.initial_dt)
```

Comment:
I am not sure this is a good idea. My reasoning behind adding the upper limit was that it should make it more safe to use (since it prevents the integrator from adapting to very large time steps by default).

The minimum step size could make it more likely that people get bad trajectories because the integrator got stuck at something like 1e-3 when the time step should have been smaller. We do print a warning, but that can be missed.

Reply:
indeed I am not sure it's a good idea to set up a minimum time step by default, what was the motivation?

Reply:
The reason is that, while in principle Damian is right and a lower bound should be very small or `None`, in practice with NQS the error estimate we have is very noisy and quickly pushes the `dt` to be something infinitesimally small in most cases I looked at, unless a very large number of samples is used.

In practice, this is... counterproductive, I think. NetKet appears unresponsive and unless you know about this, it's hard to understand if python got stuck or if simply it is using a very small dt.

My rationale was that warning about the minimum dt being reached and suggesting how to lower the bound would be better than having a hard-to-understand state.